cmd/nex/cli.go

```diff
@@ -11,10 +11,12 @@ type Globals struct {
 	GlobalLogger `prefix:"logger." group:"Logger Configuration"`
 	GlobalNats   `prefix:"nats." group:"NATS Configuration"`
 
-	Config    kong.ConfigFlag  `help:"Configuration file to load" placeholder:"./nex.config.json"`
-	Version   kong.VersionFlag `help:"Print version information"`
-	Namespace string           `env:"NEX_NAMESPACE" placeholder:"default" help:"Specifies namespace when running nex commands"`
-	Check     bool             `help:"Print the current configuration"`
+	Config              kong.ConfigFlag  `help:"Configuration file to load" placeholder:"./nex.config.json"`
+	Version             kong.VersionFlag `help:"Print version information"`
+	Namespace           string           `env:"NEX_NAMESPACE" placeholder:"default" help:"Specifies namespace when running nex commands"`
+	Check               bool             `help:"Print the current configuration"`
```

Comment:
The help text for `Check` doesn't seem accurate.

Reply:
ha...it is accurate....albeit confusing.  I blame CoPilot